lms/static/js/fixtures/search_form.html

```diff
@@ -0,0 +1,7 @@
+<div id="courseware-search-bar" class="search-container">
+    <form role="search-form">
+        <input type="text" class="search-field"/>
+        <button type="submit" class="search-button" aria-label="Search">search <i class="icon-search"></i></button>
```

Comment:
@mattdrayer, @chrisndodge, @cdodge, aria-label is unnecessary here since the text "search" is already accessible.  Suggest adding aria-hidden to the iconfont:
```<button type="submit" class="search-button">search <i class="icon-search" aria-hidden="true"></i></button>```